Documentation/helm-values.rst

```diff
@@ -265,6 +273,10 @@
      - Resource requests and limits for the clustermesh-apiserver
      - object
      - ``{}``
+   * - clustermesh.apiserver.securityContext
```

Comment:
When would we need to have this on a per-container basis for the clustermesh-apiserver?

Reply:
This might not be required, more about the consistency with other workloads.